lua/terrortown/menus/gamemode/roles/roles.lua

```diff
@@ -0,0 +1,178 @@
+--- @ignore
+
+CLGAMEMODESUBMENU.base = "base_gamemodesubmenu"
+
+CLGAMEMODESUBMENU.priority = 97
+CLGAMEMODESUBMENU.title = "submenu_administration_roles_general_title"
```

Comment:
this title identifier should be edited to match the new file location. Same holds true for the other submenus.